### ISSUE TITLE
Explore plugin installation with eclipse marketplace

### ISSUE THREAD

**Issue by yonzarecki** [TechnionYP5777/Bugquery#107]
We want the user to be able to install our plugin using the eclipse marketplace.

In this issue we'll explore how to do just that, at the end of it we should have the knowledge for implementing the eclipse installation and a new issue for the implementation.

**Comment by yonzarecki** [comment]
As we discussed in our meeting, this issue is blocked by #106 as adding a plugin to eclipse requires a working server with a public ip address

**Comment by yonzarecki** [comment]
I'd say the "Explore" part of the eclipse marketplace is over, after we get our server we will continue and implement the installation.
